src/content/guides/creating-digital-tokens.mdx

```diff
@@ -28,26 +29,112 @@ It contains redemption details specific to that merchant, e.g. redeemable produc
 
 ``` mermaid
 sequenceDiagram
-  autonumber
-  Token Issuer ->> Centrapay: Create Collection
+  Token Issuer ->> Centrapay: 1. Create Collection
   Note left of Token Issuer: Confirm redeemable product<br/>IDs with merchant
-  Token Issuer ->> Centrapay: Create Redemption Condition
-  Token Issuer ->> Centrapay: Create Token
-  Token Issuer ->> Centrapay: Transfer Token
+  Token Issuer ->> Centrapay: 2. Create Redemption Condition
+  alt
+    Note right of Token Issuer: 3a. Create tokens then send to users
+    Token Issuer ->> Centrapay: Create Token
+    Token Issuer ->> Centrapay: Transfer Token
+  else
+    Note right of Token Issuer: 3b. Create tokens into user accounts
+    Token Issuer ->> Centrapay: Create Token
```

Comment:
@seliwe Suggestion: Keep the diagram simple and indicate that the transfer is the optional part (let the written context fill in the blanks). Also let's you use `autonumber` again 😄 

```mermaid
sequenceDiagram
autonumber
Token Issuer ->> Centrapay: Create Token
 
opt
     Token Issuer ->> Centrapay: Transfer Token
end
```

Reply:
Great idea cheers